crates/utils/src/error.rs

```diff
@@ -106,6 +106,7 @@ pub enum LemmyErrorType {
   PersonIsBlocked,
   DownvotesAreDisabled,
   InstanceIsPrivate,
+  #[serde(rename = "Password must be between 10 and 60 characters")]
```

Comment:
it doesn't feel like serde rename is the right thing to do here and this seems to make the code worse to me. The error enum should be something you can match by in the client exactly e.g. `if e.error === "InvalidPassword" { show(e.message) }` and not something that will change over time or be printable directly.

So it seems to me like  maybe this should rather be `InvalidPassword(String)` or `InvalidPassword(&'static str)` or there should be a separate (client-side) mapping from LemmyErrorType to a printable (and translatable) string.

Reply:
Okay I converted this to a comment instead. Also moving the issue to lemmy-ui.